src/libs/actions/IOU.js

```diff
@@ -516,15 +488,9 @@ function getMoneyRequestInformation(
         optimisticPolicyRecentlyUsedCategories = Policy.buildOptimisticPolicyRecentlyUsedCategories(iouReport.policyID, category);
     }
 
-    const optimisticPolicyRecentlyUsedTags = {};
-    const policyTags = allPolicyTags[`${ONYXKEYS.COLLECTION.POLICY_TAGS}${iouReport.policyID}`];
-    const recentlyUsedPolicyTags = allRecentlyUsedTags[`${ONYXKEYS.COLLECTION.POLICY_RECENTLY_USED_TAGS}${iouReport.policyID}`];
-
-    if (policyTags) {
-        // For now it only uses the first tag of the policy, since multi-tags are not yet supported
-        const tagListKey = _.first(_.keys(policyTags));
-        const uniquePolicyRecentlyUsedTags = recentlyUsedPolicyTags ? _.filter(recentlyUsedPolicyTags[tagListKey], (recentlyUsedPolicyTag) => recentlyUsedPolicyTag !== tag) : [];
-        optimisticPolicyRecentlyUsedTags[tagListKey] = [tag, ...uniquePolicyRecentlyUsedTags];
+    let optimisticPolicyRecentlyUsedTags = {};
```

Comment:
Nit:

```js
const optimisticPolicyRecentlyUsedTags = condition ? result : undefined
```

(this and another similar place)

Reply:
Do you want to run that logic always?

Reply:
I wanted to suggest only a stylistic change, switching from `if` to `?`/`:`.

Reply:
```js
const optimisticPolicyRecentlyUsedTags = tag ? Policy.buildOptimisticPolicyRecentlyUsedTags(iouReport.policyID, tag) : {};
```

Reply:
Done. Commits: 3182e9b, 26b7ba9.